YubiKit/YubiKit/ThirdParties/MF_Base32Additions.m

```diff
@@ -62,7 +62,7 @@ +(NSData *)dataFromBase32String:(NSString *)encoding
                 if( c == '=' ) break; // padding...
 
                 c = decodingTable[c];
-                if( c == __ ) continue;
```

Comment:
we should not modify 3rd party lib...let's do the check on side of YubiKit which uses this method